Error compiling webiew_edge.cpp

MSVC: 14.26.28801
Windows 10 SDK: 10.0.19041.0
When trying to compile with the edge flag turned on I get the following with my toolchain:
```
running: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.26.28801\\bin\\HostX64\\x64\\cl.exe" "-nologo" "-MD" "-Z7" "-Brepro" "-I" "webview.h" "-W4" "/std:c++17" "-DDEBUG" "-DUNICODE" "-FoC:\\Users\\dylan\\Documents\\Proj\\hello\\target\\debug\\build\\webview-sys-509c560245f9ab10\\out\\webview_edge.o" "-c" "webview_edge.cpp"
webview_edge.cpp
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.26.28801\include\ppltasks.h(1571): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
webview_edge.cpp(280): warning C4100: 'a': unreferenced formal parameter
webview_edge.cpp(310): warning C4456: declaration of 'w' hides previous local declaration
webview_edge.cpp(301): note: see declaration of 'w'
webview_edge.cpp(334): warning C4100: 'debug': unreferenced formal parameter
webview_edge.cpp(389): error C2065: 'single_threaded_vector': undeclared identifier
```


I was able to fix locally by adding the following include to webview_edge.cpp:
```C++
#include <winrt/Windows.Foundation.Collections.h>
```

--- a/webview-sys/webview_edge.cpp
+++ b/webview-sys/webview_edge.cpp
@@ -14,6 +14,7 @@
 #include <wingdi.h>
 #include <winrt/Windows.Foundation.h>
 #include <winrt/Windows.Web.UI.Interop.h>
+#include <winrt/Windows.Foundation.Collections.h>
 
 #pragma comment(lib, "windowsapp.lib")
 #pragma comment(lib, "user32.lib")

